onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -18,6 +18,8 @@
 
 namespace onnxruntime {
 
+void RunOnUnload(std::function<void()> function);
```

Comment:
why you need this?

Reply:
Otherwise, it won't build. Not sure why this doesn't happen at main branch but from the headers in cuda_execution_provider.h, I don't see definition or declaration of `RunOnLoad`.